make/targets/openshift/golangci-lint.mk

```diff
@@ -0,0 +1,52 @@
+include $(addprefix $(dir $(lastword $(MAKEFILE_LIST))), \
+	../../lib/golang.mk \
+	../../lib/tmp.mk \
+)
+
+GOLANGCI_LINT_VERSION ?=1.42.1
+GOLANGCI_LINT ?=$(PERMANENT_TMP_GOPATH)/bin/golangci-lint-$(GOLANGCI_LINT_VERSION)
+
+golangci_lint_downloaded_filename := golangci-lint-$(GOLANGCI_LINT_VERSION)-$(GOHOSTOS)-$(GOHOSTARCH)
+golangci_lint_dir := $(dir $(GOLANGCI_LINT))
+
+.ensure-golangci-lint:
+ifeq "" "$(wildcard $(GOLANGCI_LINT))"
+	$(info Installing golangci-lint into '$(GOLANGCI_LINT)')
+	mkdir -p '$(golangci_lint_dir)'
+	curl -s -f -L https://github.com/golangci/golangci-lint/releases/download/v$(GOLANGCI_LINT_VERSION)/$(golangci_lint_downloaded_filename).tar.gz -o "$(PERMANENT_TMP_GOPATH)/bin/$(golangci_lint_downloaded_filename).tar.gz"
+	tar -zxf "$(PERMANENT_TMP_GOPATH)/bin/$(golangci_lint_downloaded_filename).tar.gz" -C "$(PERMANENT_TMP_GOPATH)/bin"
+	mv "$(PERMANENT_TMP_GOPATH)/bin/$(golangci_lint_downloaded_filename)/golangci-lint" $(GOLANGCI_LINT)
+	rm -rf "$(PERMANENT_TMP_GOPATH)/bin/$(golangci_lint_downloaded_filename)"
+	rm -rf "$(PERMANENT_TMP_GOPATH)/bin/$(golangci_lint_downloaded_filename).tar.gz"
+	chmod +x '$(GOLANGCI_LINT)';
+else
+	$(info Using existing golangci-lint from $(GOLANGCI_LINT))
+endif
+
+
+ensure-controller-gen:
```

Comment:
why is this target here? We've got a duplicate at `make/targets/openshift/controller-gen.mk`

Reply:
:man_facepalming:  Now you know, what file I used as a template :D